app/models/step.rb

```diff
@@ -33,6 +34,10 @@ def pre_order_tree_traversal
     [self] + self.child_approvals.flat_map(&:pre_order_tree_traversal)
   end
 
+  def completer
```

Comment:
AH yes! Makes sense re: delegate. In that case, let's do with `completer_id` so we can have a `belongs_to: :completer` relation. That seems good!